src/tests/libecalc/integration/test_all_consumer_with_time_slots_models.py

```diff
@@ -46,7 +46,7 @@ def test_time_slots_with_changing_model(time_slot_electricity_consumer_with_chan
     # train-stages are also included as CONSUMER_MODELs
     model_results = result.models
     assert isinstance(model_results, list)
-    assert len(model_results) == 3
+    assert len(model_results) == 5
```

Comment:
funny that we were testing and verifying on something that was obviously wrong?

Reply:
It's not that obvious to me. A bit confusing maybe, but it makes sense to merge the results in a lot of cases. We are solving it by rewriting the yaml currently though, and that is a better approach IMO.